Source/Core/Plugins/EventProcessor/Default/20_ErrorPlugin.cs

```diff
@@ -28,6 +29,7 @@ public class ErrorPlugin : EventProcessorPluginBase {
             if (context.HasProperty("UserNamespaces"))
                 userNamespaces = context.GetProperty<string>("UserNamespaces").SplitAndTrim(',');
 
+            AddManualStackingKeyToSignatureIfExists(context);
```

Comment:
I'll update the formatting - sorry, missed that. And, yeah it makes sense for it to be a separate step. I'll try to update in the next couple days, but may not get to it till this weekend. Thanks for the feedback!